pkgs/development/misc/jupyterKernels/default.nix

```diff
@@ -0,0 +1,76 @@
+{ lib, pkgs, stdenv}:
+
+let
+
+  defaultKernelDefinition = {
+    python3 = let
+      env = (pkgs.python3.withPackages (pythonPackages: with pythonPackages; [
+              ipykernel
+              pandas
+              scikitlearn
```

Comment:
Are these Jupyter dependencies or just nice to have packages?

Reply:
ipykernel is needed, the others are just basic packages used for data science.
I was thinking of getting something inline with the jupyter dockerimages.
Base python, base data-science, data-science + spark

Reply:
Keep the default as minimal as possible.